tgui/packages/tgui/interfaces/Sensors.tsx

```diff
@@ -0,0 +1,598 @@
+          </Table.Row>
+          {data.datalink_requests.map((request) => (
+            <Table.Row key={request.name}>
+              <Table.Cell>
+                <Button
+                  content={'Accept'}
+                  onClick={() =>
+                    act('accept_datalink_requests', {
+                      accept_datalink_requests: request.ref,
+                    })
+                  }
+                />
+              </Table.Cell>
+              <Table.Cell>
+                <Button
+                  content={'Decline'}
+                  onClick={() =>
+                    act('decline_datalink_requests', {
+                      decline_datalink_requests: request.ref,
+                    })
+                  }
+                />
+              </Table.Cell>
+              <Table.Cell>{request.name}</Table.Cell>
+            </Table.Row>
+          ))}
+        </Table>
+      ) : (
+        ''
+      )}
+      {data.datalinked && data.datalinked.length ? (
+        <Table>
+          <Table.Row header>
+            <Table.Cell>Active Datalinks:</Table.Cell>
+          </Table.Row>
+          {data.datalinked.map((datalinked) => (
+            <Table.Row key={datalinked.name}>
+              <Table.Cell>{datalinked.name}</Table.Cell>
+              <Table.Cell>
+                <Button
+                  content={'Rescind'}
+                  onClick={() =>
+                    act('remove_datalink', {
+                      remove_datalink: datalinked.ref,
+                    })
+                  }
+                />
+              </Table.Cell>
+            </Table.Row>
+          ))}
+        </Table>
+      ) : (
+        ''
+      )}
+      {data.contacts &&
+      data.contacts.length &&
+      data.contacts.some((contact) => contact.can_datalink) ? (
+        <Table>
+          <Table.Row header>
+            <Table.Cell>Potential Datalinks:</Table.Cell>
+          </Table.Row>
+          {data.contacts
+            .filter((contact) => contact.can_datalink)
+            .map((contact) => (
+              <Table.Row key={contact.name}>
+                <Table.Cell>{contact.name}</Table.Cell>
+                <Table.Cell>
+                  <Button
+                    content={'Request'}
+                    onClick={() =>
+                      act('request_datalink', {
+                        request_datalink: contact.ref,
+                      })
+                    }
+                  />
+                </Table.Cell>
+              </Table.Row>
+            ))}
+        </Table>
+      ) : (
+        ''
+      )}
+    </Section>
+  );
+};
+
+const IFFSection = function (act, data: SensorsData) {
+  return (
+    <Section title="IFF Management">
+      <Table>
+        <Table.Row>
+          <Table.Cell>State:</Table.Cell>
+          <Table.Cell>
+            {data.id_on ? 'Active' : 'Inactive'}
+            {', '}
+            <Button content="on/off" onClick={() => act('toggle_id')} />
```

Comment:
same as before, make this all one button

Reply:
done